sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -25,17 +25,7 @@
 #include <thread>
 
 namespace {
-// Can be used from anywhere a little simpler
-inline void LogThis(std::string const& msg)
```

Comment:
nit: keep the helper which appends `LogMsgPrefix` centrally, to avoid having to repeat appending it everywhere.